CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## v0.5.0
+
+* New features
+  * Node host can be set to domain provided by DHCP by setting `node_host: :dhcp`
```

Comment:
"When using Erlang distribution, the node's hostname can be set to the name returned by DHCP: `node_host: :dhcp`"